runtime/src/main/cpp/Runtime.cpp

```diff
@@ -371,3 +371,17 @@ void Kotlin_Debugging_setForceCheckedShutdown(KBoolean value) {
 }
 
 }  // extern "C"
+
+MemoryState* kotlin::mm::GetMemoryState() {
```

Comment:
Maybe implement it in each memory model instead? It'll require changing `thread_local` in `ThreadRegistry.hpp` to `Node*`, but I don't think it's a problem.

Reply:
Ok, done.